schema/settings.json

```diff
@@ -1,143 +0,0 @@
-{
-  "definitions": {},
```

Comment:
>definitions [](start = 3, length = 11)

does this also imply this pr could nuke every settings.json file that exists in the repo?

Reply:
and every categoryresources.json as well?

---
In reply to: [592703524](https://github.com/microsoft/Application-Insights-Workbooks/pull/1325#discussion_r592703524) [](ancestors = 592703524)

Reply:
and if so, we also want to add pr validation that fails if those files get re-created now that they are no longer necessary.
people with PR's in progress will be affected by this one way or the other

---
In reply to: [592704061](https://github.com/microsoft/Application-Insights-Workbooks/pull/1325#discussion_r592704061) [](ancestors = 592704061,592703524)